docs/poc/poc.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: default
+title: POC migrations
+nav_order: 8
+has_children: true
+permalink: /poc
+---
+
+# POC migrations
+{: .no_toc }
+
+Spanner migration tool can two POC migrations in two ways:
+
+## Dump Files
+
+{: .highlight }
+Dump files are only supported for MySQL and PostgreSQL.
```

Comment:
and CSV

Reply:
I suggest - Native dump files of MySQL and PostgreSQL.
Along with this we support CSV  migrations with<link to how to run csv migrations>

Reply:
Done